packages/cli/src/oclif/commands/canary/create.js

```diff
@@ -32,46 +32,66 @@ class CanaryCreateCommand extends ZapierBaseCommand {
+
+    if (versionFrom === versionTo) {
+      this.error('`VERSIONFROM` and `VERSIONTO` can not be the same')
+    }
   }
 
   validatePercent(percent) {
-    if (!percent || percent < 0 || percent > 100) {
-      this.error('Percent must be between 0 and 100');
+    if (isNaN(percent) || percent < 1 || percent > 100) {
+      this.error('`PERCENT` must be a number between 1 and 100');
     }
   }
 
   validateDuration(duration) {
-    if (!duration || duration <= 30 || duration > 24 * 60 * 60) {
-      this.error('Duration must be a positive number between 30 and 86400');
+    if (isNaN(duration) || duration < 30 || duration > 24 * 60 * 60) {
+      this.error('`DURATION` must be a positive number between 30 and 86400');
     }
   }
 }
 
-CanaryCreateCommand.flags = buildFlags({
-  commandFlags: {
-    versionFrom: flags.string({char: 'f', description: 'Version to route traffic from', required: true}),
-    versionTo: flags.string({char: 't', description: 'Version to canary traffic to', required: true}),
-    percent: flags.integer({char: 'p', description: 'Percent of traffic to route to new version', required: true}),
-    duration: flags.integer({char: 'd', description: 'Duration of the canary in seconds', required: true}),
+CanaryCreateCommand.args = [
+  {
+    name: 'versionFrom',
+    required: true,
+    description: 'Version to route traffic from',
   },
-  opts: {
-    format: true
-  }
-});
+  {
+    name: 'versionTo',
+    required: true,
+    description: 'Version to canary traffic to',
+  },
+  {
+    name: 'percent',
+    required: true,
+    description: 'Percentage (between 1 and 100) of traffic to route to new version',
+    parse: (input) => parseInt(input, 10),
+  },
+  {
+    name: 'duration',
+    required: true,
+    description: 'Duration (between 30 and 86400) of the canary in seconds',
+    parse: (input) => parseInt(input, 10),
+  },
+];
+
 CanaryCreateCommand.description =
-  'Create a new canary deployment, diverting a specified percentage of traffic from one version to another for a specified duration.'
+  'Create a new canary deployment, diverting a specified percentage of traffic from one version to another for a specified duration. \n' +
+  '\n' +
+  'Only one canary can be active at the same time. You can run `zapier canary:list` to check. If you would like to create a new canary with different parameters, you can wait for the canary to finish, or delete it using `zapier canary:delete a.b.c x.y.z`. \n' +
+  '\n' +
+  'Note: this is similar to `zapier migrate` but different in that this is temporary and will "revert" the changes once the specified duration is expired. \n' +
+  '\n' +
+  '**Only use this command to canary traffic between non-breaking versions!**';
+
 CanaryCreateCommand.examples = [
-  'zapier canary:create --versionFrom=1.0.0 --versionTo=1.1.0 --percent=25 --duration=720',
-  'zapier canary:create -f 2.0.0 -t 2.1.0 -p 50 -d 300'
-]
+  'zapier canary:create 1.0.0 1.1.0 25 720',
```

Comment:
Idk if this is a problem or not, but the percent and duration numbers can get mixed up easily. It'd be bad if they wanted 100s of canary and 30% traffic, but it was in the wrong order, creating a canary that is 100% traffic for 30s. 

So I think it'd be nice to either 
1. Show the summary upfront but ask for confirmation
2. turn percent/duration into options (so it can be passed in like -p 25, -d 30 for example) 

I'm fine with either option